packaging/pep517_backend/_cython_configuration.py

```diff
@@ -79,10 +79,18 @@ def get_local_cython_config() -> Config:
     return config_mapping['tool']['local']['cythonize']  # type: ignore[no-any-return]
 
 
-def make_cythonize_cli_args_from_config(config: Config) -> list[str]:
+def make_cythonize_cli_args_from_config(config: Config, cython_line_tracing_requested: bool = False) -> list[str]:
     py_ver_arg = f'-{_python_version_tuple.major!s}'
 
     cli_flags = get_enabled_cli_flags_from_config(config['flags'])
+
+    # If line tracing is requested, add it to the directives
+    if cython_line_tracing_requested:
+        directives = config['kwargs'].setdefault('directive', {})
```

Comment:
Can we move `config['kwargs']` into a dedicated var, as it's also used in the return statement?

Reply:
It mutating the `dict` so it will be captured in the return statement

Reply:
Yes, that's what I was thinking.